packages/styletron-engine-atomic/src/inject-style-prefixed.ts

```diff
@@ -77,13 +77,16 @@ export default function injectStylePrefixed(
             media,
             pseudo + originalKey,
           );
-      } else if (originalKey.substring(0, 6) === "@media") {
+      } else if (
+        originalKey.substring(0, 6) === "@media" ||
+        originalKey.substring(0, 10) === "@container"
+      ) {
         classString +=
           " " +
           injectStylePrefixed(
             styleCache,
             originalVal as StyleObject,
-            originalKey.substr(7),
+            originalKey,
```

Comment:
To be honest, I don't remember the original reasoning for not storing the at-prefix. But looking at this now, this seems preferable since it avoids having to get the substring and also avoids the later concatenation upon injection

Reply:
Great! I considered just dropping the substring matching entirely and just checking if the first character was `@`, but that seemed beyond the scope of this change, but might be worth considering at some point.